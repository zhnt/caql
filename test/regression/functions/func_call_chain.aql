function level3() {
    print("Level 3 function")
}

function level2() {
    print("Level 2 start")
    level3()
    print("Level 2 end")
}

function level1() {
    print("Level 1 start")
    level2()
    print("Level 1 end")
}

print("Main start")
level1()
print("Main end")
