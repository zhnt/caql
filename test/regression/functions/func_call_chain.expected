Main start
Level 1 start
Level 2 start
Level 3 function
Level 2 end
Level 1 end
Main end

